content/forking-a-core-package/sections/creating-a-fork-of-a-core-package-in-atom-atom.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Creating a Fork of a Core Package in atom/atom
+---
+### Creating a Fork of a Core Package in atom/atom
+
+Several of Atom's core packages are maintained in the [`packages` directory of the atom/atom repository](https://github.com/atom/atom/tree/master/packages). If you would like to use one of these packages as a starting point for your own package, please follow the steps below.
+
+#### Creating Your New Package
+
+For the sake of this guide, let's assume that you want to start with the current code in the [one-light-ui](https://github.com/atom/atom/tree/master/packages/one-light-ui) package, make some customizations to it, and publish your new package under the name "one-light-ui-plus".
+
+1. Download the [current contents of the atom/atom repository as a gzipped tarball](https://github.com/atom/atom/archive/master.tar.gz)
+
+2. Unzip the file to a temporary location (for example `/tmp/atom`)
+
+3. Copy the contents of the desired package into a working directory for your fork
+
+    ``` command-line
+    $ cp -R /tmp/atom/packages/one-light-ui ~/src/one-light-ui-plus
+    ```
```

Comment:
I also tend to avoid ordered lists for instructions because the OS-specific blocks (spans are fine though) don't interact well with ordered lists. And I tend to prefer to have OS-specific blocks around sections like this where the Unix-style and Windows-style commands are very different.